clustered/client/src/main/resources/ehcache-clustered-ext.xsd

```diff
@@ -157,13 +158,23 @@
         </xs:annotation>
       </xs:element>
     </xs:sequence>
-    <xs:attribute name="auto-create" type="xs:boolean" use="optional" default="false">
```

Comment:
why don't we keep this to let the user know what's used by default ?

Reply:
Sadly the necessary information associated with the DOM that distinguishes between the 'default' value and a value equal to it is elided by the JAXB parser. This means if these attributes have default values in the XSD we can't tell if the user actually defined a value at all.